modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -22,6 +22,16 @@ def fetch_aud
 
       protected
 
+      def validate_veteran_identifiers
+        return if target_veteran.participant_id.present? && target_veteran.birls_id.present?
+        return unless header_request?
```

Comment:
could this be an `if` condition in the `before_action`?

Reply:
and then could just combine the other returns into
```ruby
return if (target_veteran.participant_id.present? && target_veteran.birls_id.present?) || target_veteran.mpi_record?
```

Reply:
i see where your mind is at but this method gets more complicated in subsequent prs around this logic. what you described might work here, but i dont believe it will work down the line